en/02_Developer_Guides/17_CLI/01_Sake.md

```diff
@@ -95,7 +95,7 @@ sake <command> --help # e.g. `sake db:build --help`
 ```
 
 > [!CAUTION]
-> You should run `sake` with the same system user that runs your web server. Otherwise you will have a separate filesystem cache for CLI and you won't be able to flush or warm your webserver cache using Sake.
+> You should run `sake` with the same system user and PHP version that runs your web server. Otherwise you will have a separate filesystem cache for CLI and you won't be able to flush or warm your webserver cache using Sake. Other unexpected behaviour could also occur e.g. as a result of different permissions.
```

Comment:
Addad PHP version for completeness, and added note about other unexpected behaviour because that's probably more important than just "you can't warm your cache"